src/modules/ipops/doc/ipops_admin.xml

```diff
@@ -883,6 +883,57 @@ if(dns_query("test.com", "xyz"))
 
     </section>
 
+    <section id="ipops.f.ptr_query">
+      <title>
+        <function moreinfo="none">ptr_query(ip, pvid)</function>
+      </title>
+
+      <para>
+		  Store the hostname that correspond to ip
+		  in a config variable $ptrquery(pvid=>hostname).
+      </para>
+
+      <para>Parameters:</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>
+			  <emphasis>ip</emphasis> - string or pseudo-variable containing the ip.
+			  The resulting IP addresses from DNS query are compared with ipaddr.
```

Comment:
In line 901 you just say "ip" and here "ipaddr". I suggest you write clear text on both cases. Like "containing the IP address" and "compared with the IP address"

Reply:
Removed altogether. there is no comparison done at all. It must be a leftover from DNS query.